### PR TITLE
Wrote about the subject.

### PR DESCRIPTION
**Pull request by LucasJSch**

### CODE REVIEW

**Comment by LucasJSch** [comment]
Wrote different sections. Changes may be made during the week.

**Comment by ernestmc** [comment]
Please use a descriptive title when creating a pull-request. It should summarize the purpose of the PR.
Also you have conflicts in the file. You should pull the latest changes from master branch and rebase your working branch, solving any conflicts in the way.

File: lucas_scheinkerman/README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+
+
+-Besides the angular velocity of the wheels, there are two more factors that define the pose variation (position and direction) of the robot. These are the **wheels radius** and the **distance between the two wheels**. 
+Intuitively, its easy to see that the longer the distance between the wheels, greater angular velocities of the wheels will be needed for getting the same velocity of the robot as with shorter wheel distance. This is because as the wheels distance becomes larger, the circumference which will be involved in the rotational velocity of the robot will be bigger, which makes the rotational velocity of the robot lower.
+With the wheels radius the opposite effect takes place. As the radius becomes bigger, with constant wheel angular velocity, the velocity of the robot increases. 
+
+
+- Mathematically, all that we explained before can be described with the following equations:
```
Comment (ernestmc) [inline lucas_scheinkerman/README.md:19]: A graphic of all these magnitudes can help a lot here.

**Review by ernestmc — COMMENTED** [review]
Looks good. Left some comments

**Pull request closed without merging**